tests/test_aggregate.py

```diff
@@ -10,6 +10,12 @@
 
 
 class TestClimatologicalMean:
+    def _format(self, s):
```

Comment:
Yes.
In climatological_op the generation of the description and history use the formatter. Since climatological_mean now calls climatological_op description and history are according to climatological_op, and the tests were adjusted for that. Unless the tests could call TestClimatologicalOp._format().

Reply:
Done.